examples/primitives-torus/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
we can get rid of blank lines, and indent properly by 2 spaces. check the other example HTML files